chia/harvester/harvester_api.py

```diff
@@ -108,7 +111,7 @@ def blocking_lookup(filename: Path, plot_info: PlotInfo) -> list[tuple[bytes32,
                     new_challenge.sp_hash,
                 )
                 try:
-                    quality_strings = plot_info.prover.get_qualities_for_challenge(sp_challenge_hash)
+                    quality_strings = plot_info.prover.get_qualities_for_challenge(sp_challenge_hash, plot_strength)
```

Comment:
im not sure that we would want the filtering done in the prover and not in chia-blockchain, unless we expect to wrap the v2prover like we did with v1 then we can filter in the wrapper, that would be ok i thinkn

Reply:
we could wrap it in a single method and keep the filtering logic in one place but we dont decided to not be aware of plot strength in chia blockchain so we can close this

Reply:
> do we even want get_qualities_for_challenge to work for v2 ?

I think that's reasonable, at least for now. As I have fixed support for v2 plots in `check_plots` I'm increasingly thinking that we probably won't have a need for this on v2 plots long term. But this protocol is also having some problems with the abstraction it's attempting. So on the one hand I think it's good to have the interface work the same. But to get there we may also need to make v1 plots have a more similar flow as v2 plots do.

Reply:
I restored this